rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -450,6 +450,17 @@ public enum BridgeMethods {
             activations -> activations.isActive(RSKIP134),
             fixedPermission(true)
     ),
+    GET_ACTIVE_POWPEG_REDEEM_SCRIPT(
+            CallTransaction.Function.fromSignature(
+                    "getActivePowpegRedeemScript",
+                    new String[]{},
+                    new String[]{"bytes"}
+            ),
+            fixedCost(3_000L), // TODO: DEFINE THE CORRECT FIXED COST FOR THS METHOD
+            (BridgeMethodExecutorTyped) Bridge::getActivePowpegRedeemScript,
+            activations -> activations.isActive(RSKIP293),
+            fixedPermission(false)
```

Comment:
Why isn't this local only? AFAIK is for external use, so it could be limited to that.

Reply:
This is doing a simple operation that poses no risk of misuse of the node resources. Thus, we understand this is not needed.